src/App.tsx

```diff
@@ -1,26 +1,173 @@
-/* eslint-disable max-len */
+/* eslint-disable react-hooks/rules-of-hooks */
+/* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
-
-const USER_ID = 0;
+import React, { useEffect, useState } from 'react';
+import { USER_ID, deleteTodo, getTodos } from '../src/components/api/todos';
+import { TodoList } from './components/ToDoList';
+import { TodoHeader } from './components/ToDoHeader';
+import { TodoFooter } from './components/ToDoFooter';
+import { Error } from './components/Error';
+import { type Todo } from './types/Todo';
+import { createTodo } from '../src/components/api/todos';
+import { FilterButtons } from './types/FilterType';
+import { Errors } from './types/EnumedErrors';
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
-  }
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [error, setError] = useState<Errors>(Errors.NoLetters);
+  const [temporaryTodo, setTemporaryTodo] = useState<Todo | null>(null);
+  const [tempIds, setTempIds] = useState<number[]>([]);
+  const [newToDoTitle, setNewToDoTitle] = useState('');
+  const [filterButton, setFilterButton] = useState<FilterButtons>(
+    FilterButtons.All,
+  );
+  const [isLoading, setIsLoading] = useState(false);
+
+  const handleError = (errorsTexts: Errors) => {
+    setError(errorsTexts);
+  };
+
+  useEffect(() => {
+    getTodos()
+      .then(setTodos)
+      .catch(() => {
+        setError(Errors.UnableToLoad);
+        setTimeout(() => {
+          setError(Errors.NoLetters);
+        }, 3000);
+      });
+  }, [error]);
```

Comment:
you shouldn't getTodos every time error changes because then you have loop of errors if you can't load todos